### ISSUE TITLE
Using version 0.2.2b, game is using 89-99% CPU and not responding.

### ISSUE THREAD

**Issue by Konuchi** [Kaldaien/UnX#24]
With 0.2.2b, upon launching the game, the window stays black and the title says it is not responding.  The task manager shows the game using 89-99% of my CPU.  Previous versions of UnX did not do this.  Switching back to 0.2.1 fixes it.

**Comment by Kaldaien** [comment]
Please update to 0.2.3 and comment on whether it was fixed.

**Comment by Konuchi** [comment]
It was fixed in 0.2.3 but it is happening again with 0.3.0.

Setting

```
[UnX.Stutter]
Reduce=true
```

To false seems to fix it in 0.3.0.

**Comment by Kaldaien** [comment]
Yeah, that was noted in the release notes. I've added information on configuring that to the Steam guide itself too.

High CPU load isn't necessarily bad, particularly since poor GPU performance in this game is often caused by ineffective CPU load balancing (too conservative). If you have a lower-end CPU though, disable the feature.
